src/commands/auth/sfdxurl/store.ts

```diff
@@ -64,4 +69,15 @@ export default class Store extends SfdxCommand {
     this.ux.log(successMsg);
     return result;
   }
+
+  private async getUrlFromJson(authFile: string): Promise<string> {
+    const authFileJson = (await fs.readJson(authFile)) as Record<string, string>;
```

Comment:
You could clean this up a bit by adding this type:

```typescript
type AuthJson = AnyJson & {
  result? : AnyJson & { sfdxAuthUrl: string };
  sfdxAuthUrl: string;
};
```

I'm pretty sure that would eliminate the need to do all the casting